src/source/publishing/influx_publisher.cr

```diff
@@ -79,6 +82,7 @@ module PlaceOS::Source
       end
       tags["pos_system"] = data.control_system_id
       tags["pos_index"] = data.index.to_i64.to_s
+      tags["mod_name"] = data.module_name
```

Comment:
Should we keep our fields `pos_` prefixed for consistency?

Reply:
yeah good idea